lib/frequencyStats.js

```diff
@@ -1,20 +1,19 @@
 function frequencyStats(array) {
-    let counts = {};
-    let charCount = array.length
+  let counts = {};
+  let charCount = array.length;
 
-    for (let item of array.sort()) {
-      if (!counts[item]) {
-        counts[item] = 1;
-      } else {
-        counts[item] += 1;
-      }
+  for (let item of array.sort()) {
```

Comment:
This is `itemCounts`! Require it and re-use it here. :)

Also, I probably wouldn't sort the keys here. That's really a display concern. I'd sort it at display time. Imagine if you wanted to sort the letters by frequency or something else.